src/test/kotlin/uk/gov/justice/digital/hmpps/hmppschallengesupportinterventionplanapi/controller/SearchCsipRecordsIntTest.kt

```diff
@@ -403,4 +417,16 @@ class SearchCsipRecordsIntTest : IntegrationTestBase() {
     assertThat(lastName).isEqualTo(person.lastName)
     assertThat(location).isEqualTo(person.cellLocation)
   }
+
+  private fun matchesPrison(vararg prisonCodes: String) = Specification<CsipRecord> { csip, _, cb ->
+    val person = csip.join<CsipRecord, PersonSummary>("personSummary")
+    cb.or(person.get<String>("prisonCode").`in`(prisonCodes.toSet()), person.get<String>("supportingPrisonCode").`in`(prisonCodes.toSet()))
+  }
+
+  private fun clearDown(vararg prisonCodes: String) = transactionTemplate.executeWithoutResult {
```

Comment:
Was this clear down function needed for testing the new plan properties or is it to clean up data from other tests?